theforeman.org/yaml/publishers/ircbot_freenode.yaml

```
@@ -29,3 +29,9 @@
             - name: '#theforeman-dev'
               notify-only: true
         matrix-notifier: only-configurations
+
+- publisher:
+    name: email_list
```

I'd prefer to use https://community.theforeman.org/c/infra/21
```suggestion
    name: discourse
```